src/demux.js

```diff
@@ -45,10 +40,11 @@ class DemuxSink extends Writable {
      * @param {boolean} persistDump - Flag used for generating a complete dump of the data coming to the stream.
      * Required when creating mock tests.
      */
-    constructor({ tempPath, dumpFolder, connectionInfo, log, persistDump = false }) {
+    constructor({ tempPath, dumpFolder, dumpPath, connectionInfo, log, persistDump = false }) {
```

Comment:
I think tempPath and dumpFolder are no longer needed if we're going to rely on the newly added dumpPath right?